pkg/informer/reflector.go

```diff
@@ -0,0 +1,117 @@
+// Copyright 2023 TiKV Project Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package informer
+
+import "github.com/pingcap/kvproto/pkg/pdpb"
+
+type reflector struct {
```

Comment:
There are expect defferent refelctors for `region item` and `store item`? and different implement store?  

BTW, how to get region,store information?